### PR TITLE
Update to toast list to set appropriate width

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Also:
Fix toasts inside toast list so an extra div is not needed.
Remove the toast notifications list test page and update the toast
notifications test page to use the list.
Use toast notification list on full page layout test pages for example
warning text.

### CODE REVIEW

**Comment by dgutride** [comment]
This looks good to me.

**Comment by jeff-phillips-18** [comment]
Pushed minor updates to use grid-gutter-width / 2 rather than hard coded left margin for the toast list and to remove the left margin when not in mobile mode.

File: less/toast.less [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // --------------------------------------------------
 
 .toast-pf {
-  background-color: hsla(0, 0%, 100%, .94);
+  background-color: hsla(0, 0%, 100%, 1);
```
Comment (andresgalante) [inline less/toast.less]: @jeff-phillips-18 please use the variable white.

File: less/toast.less [hunk 2]
```diff
@@ -51,11 +51,21 @@
 }
 
 .toast-notifications-list-pf {
+  margin-left: (@grid-gutter-width / 2);
   position: fixed;
   right: 15px;
```
Comment (andresgalante) [inline less/toast.less]: same here, gutter/2 for right and top

File: less/toast.less [hunk 3]
```diff
@@ -3,7 +3,7 @@
 // --------------------------------------------------
 
 .toast-pf {
-  background-color: hsla(0, 0%, 100%, .94);
+  background-color: hsla(0, 0%, 100%, 1);
   border-color: @color-pf-black-400;
   box-shadow: 0 2px 6px hsla(0, 0%, 0%, .2);
```
Comment (andresgalante) [inline less/toast.less]: please use a less fade http://lesscss.org/functions/#color-operations-fade of the white variable

File: less/toast.less [hunk 4]
```diff
@@ -51,11 +51,21 @@
...
   position: fixed;
   right: 15px;
   top: 15px;
   z-index: 1050;
   .toast-pf {
+    clear: both;
     float: right;
   }
+  @media (min-width: @screen-sm-min) {
+    margin-left: 0;
```
Comment (andresgalante) [inline less/toast.less]: now that I am thinking, if the width is 50% then it doesn't matter if we leave the margin left, right?

**Comment by andresgalante** [comment]
Thanks @jeff-phillips-18 !

File: less/toast.less (hunk 4 above)
Comment (jeff-phillips-18) [inline less/toast.less]: correct. I will remove this.

File: less/toast.less (hunk 3 above)
Comment (jeff-phillips-18) [inline less/toast.less]: Changing this to straight @color-pf-white without any fading per comment above. This was really unnecessary once I changed the opacity to 1. Sorry about that.

**Comment by jeff-phillips-18** [comment]
Addressed @andresgalante  issues.

**Pull request merged**